prepare_offline_installation/prepare_offline_installation.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+# simplify your orcharhino Server offline installation by generating commands based on a list of repositories
+
+###################################################################
+# adapt path to your offline medium, for example an external disk
+repo_path = "/media/my_disk/my_repos/"
+# adapt list of repositories
+repositories = [
+    "Insert list from ATIX Service Portal:",
+    "https://atixservice.zendesk.com/hc/de/articles/4414413251346",
+]
+###################################################################
+
+# create repo file
+repo_file = "/tmp/orcharhino.repo"
+repo_file_content = """"""
+skeleton = """
+[REPO_ID]
+name=REPO_ID
+baseurl=file://REPO_PATH
+enabled=1
+gpgcheck=0
+"""
+for repository in repositories:
+    repo_file_content = repo_file_content + skeleton.replace("REPO_ID", repository).replace("REPO_PATH", repo_path + repository)
```

Comment:
You can use `repo_file_content += skeleton...` here.
Also applies to the two other lines that are similar.